README.md

```diff
@@ -66,11 +40,48 @@ Build a small application exposing the following endpoints through a REST API:
 
 We'll evaluate:
  
- * the complexity & scalability of your algorithm
- * the readability of your code (including readability of your tests)
- * the correctness of the outputs of the API
- * you're ability to share your design choices and clearly weight pros & cons of alternative solutions
- * the overall quality of your written technical communication 
+ * The complexity & scalability of your algorithm
+ * The readability of your code (including readability of your tests)
+ * The correctness of the outputs of the API
+ * You're ability to share your design choices and clearly weight pros & cons of alternative solutions
+ * The overall quality of your written technical communication
+
+
+### Part 2: System design
+
+Describe briefly how you would build at scale and in the cloud (preferably with GCP) an API to power a simplified page of our Search Analytics:
```

Comment:
If we want to judge them mainly on that part, I'd recommend removing "briefly"
```suggestion
Describe how you would build at scale and in the cloud (preferably with GCP) an API to power a simplified page of our Search Analytics:
```

Reply:
We want them to be concise and not handling us tens of pages of specification (that's why we added a size limit). The goal here is to use their design description as a base for our discussion during the assignment review where we can challenge it/add further constraints to see how they adapt (like we do during a regular architecture design specification)

Reply:
Let's go with something like "at most a page of text and as many diagrams as you deem necessary for the good comprehension of your design."